### ISSUE TITLE
Adopt Styled Components improvements

### ISSUE THREAD

**Issue by ashfurrow** [Moya/contributors#19]
styled-components is a really popular JS org that uses the Moya contributor guidelines as a base for their own. They [recently updated their guidelines](https://github.com/styled-components/styled-components/pull/1364) and we investigate to see about borrowing ideas back into our own contribute guidelines.

**Comment by BasThomas** [comment]
Awesome, thanks for bringing this up @ashfurrow!

Although I will take some time to let everything I just read sink in, I opened a PR with some fixes (and thoughts) [over at Styled Components](https://github.com/styled-components/styled-components/pull/1369).

**Comment by mxstbr** [comment]
Note: Did some more updates today too, make sure you refernence the newest version of the CONTRIBUTING.md/CORE_TEAM.md. https://github.com/styled-components/styled-components/pull/1366

**Comment by BasThomas** [comment]
Yeah I saw that, was a bit confusing reading through the commit mentioned by Ash and my local copy of `master`. 😁
